use-timescale/extensions/index.md

```diff
@@ -13,50 +13,120 @@ supported extensions:
+| [fuzzystrmatch][fuzzystrmatch]                   | Determine similarities and distance between strings                    | -                              |
+| [hstore][hstore]                                 | Data type for storing sets of (key, value) pairs                       | -                              |
+| [hstore_plperl][hstore]                          | Transform between hstore and plperl                                    | -                              |
+| [insert_username][insert_username]               | Functions for tracking who changed a table                             | -                              |
+| [intagg][intagg]                                 | Integer aggregator and enumerator (obsolete)                           | -                              |
+| [intarray][intarray]                             | Functions, operators, and index support for 1-D arrays of integers     | -                              |
+| [isn][isn]                                       | Data types for international product numbering standards               | -                              |
+| [jsonb_plperl][jsonb_plperl]                     | Transform between jsonb and plperl                                     | -                              | 
+| [lo][lo]                                         | Large object maintenance                                               | -                              |
+| [ltree][ltree]                                   | Data type for hierarchical tree-like structures                        | -                              |
+| [moddatetime][moddatetime]                       | Functions for tracking last modification time                          | -                              |
+| [old_snapshot][old_snapshot]                     | Utilities in support of `old_snapshot_threshold`                       | -                              |
+| [pg_freespacemap][pg_freespacemap]               | Examine the free space map (FSM)                                       | -                              |
+| [pg_prewarm][pg_prewarm]                         | Prewarm relation data                                                  | -                              |
+| [pg_stat_statements][pg_stat_statements]         | Track execution statistics of all SQL statements executed              | All                            |
+| [pg_trgm][pg_trgm]                               | Text similarity measurement and index searching based on trigrams      | -                              |
+| [pg_visibility][pg_visibility]                   | Examine the visibility map (VM) and page-level visibility info         | -                              |
+| [pgai][pgai]                                     | Helper functions for AI workflows                                      | All                            |
```

Comment:
it's called `ai` in the extension list

Reply:
I know, but it's called pgai in docs, so I am using this name for consistency.